manifests/join.pp

```diff
@@ -51,7 +57,7 @@
     exec {'adcli_join':
       command   => "/bin/echo -n \'${ad_join_password}\' | /usr/sbin/adcli join ${ad_join_domain_controller_command} --login-user=\'${ad_join_username}\' \
 --domain=\'${ad_domain}\' --domain-ou=\'${ad_join_ou}\' --stdin-password --verbose ${ad_join_os_command} \
-${ad_join_os_version_command} ${ad_join_os_service_pack_command}",
+${ad_join_os_version_command} ${ad_join_os_service_pack_command} ${ad_join_service_names_comand}",
```

Comment:
please correct and spell command properly

Reply:
comand has been corrected to command